vzstorage-pd/virtuozzo-provisioner.go

```diff
@@ -284,7 +283,7 @@ func (p *vzFSProvisioner) Delete(volume *v1.PersistentVolume) error {
 var (
 	master        = flag.String("master", "", "Master URL")
 	kubeconfig    = flag.String("kubeconfig", "", "Absolute path to the kubeconfig")
-	provisionerID = flag.String("name", "", "Unique provisioner name")
+	provisionerID = flag.String("name", "virtuozzo.com/virtuozzo-storage", "Unique provisioner name")
```

Comment:
provisionerID allows to have a few instances of one provisioner (both instances use the same  provisionerName.
Don't mix these terms. You have to add a separate options for provisionerName.